### PR TITLE
Play 2.7.0 RC3

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,9 +4,17 @@ version := "1.0-SNAPSHOT"
...
-scalaVersion := "2.12.6"
+scalaVersion := "2.12.7"
 
-crossScalaVersions := Seq("2.11.12", "2.12.4")
+javacOptions ++= Seq(
+  "-encoding", "UTF-8",
+  "-parameters",
+  "-Xlint:unchecked",
+  "-Xlint:deprecation",
+  "-Werror"
```
Comment (marcospereira) [inline build.sbt:14]: To catch any use of deprecated APIs in our sample apps.

File: .travis.yml [hunk 2]
```diff
@@ -27,21 +25,20 @@ before_cache:
...
 # https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix
 matrix:
-  exclude:
-    - scala: 2.11.12
-      jdk: oraclejdk9
-    - scala: 2.11.12
-      jdk: oraclejdk10
-    - scala: 2.11.12
-      jdk: oraclejdk11
+  include:
```
Comment (marcospereira) [inline .travis.yml:28]: We should later [use jabba](http://eed3si9n.com/all-your-jdks-on-travis-ci-using-jabba) to install AdoptOpenJDK instead.

**Comment by marcospereira** [comment]
@playframework/core this can be merged as is since AdoptOpenJDK 8 was removed from Jabba (see https://github.com/shyiko/jabba/pull/338 and https://github.com/shyiko/jabba/pull/340). We can later use AdoptOpenJDK instead of OpenJDK.

**Review by dwijnand — APPROVED** [review]
SGTM

**Pull request merged**